contracts/6/forwarder/ERC20Forwarder.sol

```diff
@@ -0,0 +1,264 @@
+pragma solidity 0.6.9;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./ERC20ForwarderStorage.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "../interfaces/IFeeManager.sol";
+import "./BiconomyForwarder.sol";
+
+
+/**
+ * @title ERC20 Forwarder
+ *
```

Comment:
Mention that this follows upgradable pattern.

Reply:
done